etna/models/mixins.py

```diff
@@ -348,6 +414,33 @@ def _make_predictions(self, ts: TSDataset, prediction_method: Callable, **kwargs
             ts.df = ts.df.iloc[-prediction_size:]
         return ts
 
+    def _make_component_predictions(self, ts: TSDataset, prediction_method: Callable, **kwargs) -> pd.DataFrame:
```

Comment:
Why do we need this method? How is it different from `_make_predictions`?

Reply:
Here we create dataframe with components and in `_make_predictions` we create dataset with forecast. There is some difference in logic